code/datums/outfits/outfit_admin.dm

```diff
@@ -849,10 +847,94 @@
 	backpack_contents = list(
 		/obj/item/storage/box/survival = 1,
```

Comment:
use the box variables for all of these, so you dont have to change the bag
`/obj/item/storage/box/survival`